pkgs/by-name/op/optar/package.nix

```diff
@@ -31,11 +31,17 @@ stdenv.mkDerivation {
     mkdir -p $out/bin
   '';
 
+  # gcc14 is more strict
+  env.NIX_CFLAGS_COMPILE = toString [
+    "-Wno-error=implicit-int"
+  ];
```

Comment:
Let’s apply a proper fix instead; [this AUR comment](https://aur.archlinux.org/packages/optar#comment-991174) has a patch.